src/components/proposal/ProposalTableUser.tsx

```diff
@@ -53,13 +55,21 @@ const ProposalTableUser: React.FC = () => {
               );
             })
             .map((proposal) => {
+              const hasReferenceNumberFormat = proposal.call
+                ?.referenceNumberFormat
```

Comment:
Here also using `!!roposal.call?.referenceNumberFormat` will make it a bit cleaner.